human-kinases/human_kinases.ipynb

```diff
@@ -0,0 +1,3599 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "import urllib.request\n",
+    "\n",
+    "import requests\n",
+    "import bs4\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "REPO = (Path(_dh[-1]) / \"..\").resolve()\n",
+    "DATA = REPO / 'data'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Obtain UniprotIDs for all human kinases\n",
+    "\n",
+    "There are several sources that can be used to identify the human kinome in Uniprot:\n",
```

Comment:
Update list of sources. Add numbers and use throughout the nb, makes it easier to follow.

Reply:
use numbers with or without point consistently throughput the nb. 
Eg. currently: 01. KinHub, 02 KLIFS, ...